packages/flutter/lib/src/rendering/flex.dart

```diff
@@ -486,7 +486,7 @@ class RenderFlex extends RenderBox with ContainerRenderObjectMixin<RenderBox, Fl
   double? _overflow;
   // Check whether any meaningful overflow is present. Values below an epsilon
   // are treated as not overflowing.
-  bool get _hasOverflow => _overflow! > precisionErrorTolerance;
+  bool get _hasOverflow => _overflow != null && _overflow! > precisionErrorTolerance;
```

Comment:
If we treat `_overflow == null` as "no overflow", why not just assign `_overflow` an initial value of false and make it non-nullable?

Reply:
Good point - done.